backend/src/routes/api/k8s/pass-through.ts

```diff
@@ -9,7 +9,7 @@ import { proxyCall, ProxyError, ProxyErrorType } from '../../../utils/httpUtils'
 
 export type PassThroughData = {
   method: string;
-  requestData: string;
+  requestData?: string;
```

Comment:
🤔 intentional change? I guess for pure query params?

Reply:
This aligns with the types for `proxyCall` it's optional. I have no data for a `GET` request.